packages/tooltip/tooltip.stories.tsx

```diff
@@ -0,0 +1,159 @@
+          ref: tooltipRef,
+          style: styles
+        })}
+      >
+        Tooltip
+      </div>
+    </>
+  );
+};
+
+export const WithPopper = () => {
+  const tooltipRef = useRef(null);
+  const triggerRef = useRef(null);
+
+  const { isVisible, getTooltipProps, getTriggerProps } = useTooltip({
+    isVisible: boolean('isVisible', false),
+    delayMilliseconds: number('Tooltip delay', 500)
+  });
+  const { style } = usePopper({ referenceRef: triggerRef, popperRef: tooltipRef });
+
+  const styles: CSSStyleDeclaration | React.CSSProperties = {
+    ...style,
+    visibility: isVisible ? 'visible' : 'hidden',
+    background: '#1f73b7',
+    padding: '10px',
+    margin: '6px 0',
+    color: '#fff'
+  };
+
+  return (
+    <>
+      <div {...getTooltipProps({ ref: tooltipRef, style: styles })}>Tooltip</div>
+      <button {...(getTriggerProps({ ref: triggerRef }) as any)}>Trigger</button>
+    </>
+  );
+};
+
+export const FocusableTooltip = () => {
+  const tooltipRef = useRef(null);
+
+  const { isVisible, getTooltipProps, getTriggerProps, openTooltip, closeTooltip } = useTooltip({
+    isVisible: boolean('isVisible', false),
+    delayMilliseconds: number('Tooltip delay', 500)
+  });
+
+  const styles: React.CSSProperties = {
+    visibility: isVisible ? 'visible' : 'hidden',
+    background: '#000',
+    padding: '10px',
+    margin: '6px 0',
+    color: '#fff'
+  };
+
+  return (
+    <>
+      <button {...getTriggerProps()}>Trigger</button>
+      <div
+        {...getTooltipProps({
+          ref: tooltipRef,
+          style: styles,
+          onFocus: () => openTooltip(),
+          onBlur: () => closeTooltip(0)
+        })}
+      >
+        Focusable tooltip, tab to this <button>button</button>
+      </div>
+    </>
+  );
+};
+
+export const Container = () => {
+  const tooltipRef = useRef(null);
+
+  return (
+    <TooltipContainer
+      isVisible={boolean('isVisible', false)}
+      delayMilliseconds={number('Tooltip delay', 500)}
+    >
+      {({ isVisible, getTooltipProps, getTriggerProps }) => {
+        const styles: React.CSSProperties = {
+          visibility: isVisible ? 'visible' : 'hidden',
+          background: '#1f73b7',
+          padding: '10px',
+          margin: '6px 0',
+          color: '#fff'
+        };
+
+        return (
+          <>
+            <div
+              {...getTooltipProps({
+                ref: tooltipRef,
+                style: styles
+              })}
+            >
+              Tooltip
+            </div>
+            <button {...getTriggerProps()}>Trigger</button>
+          </>
+        );
+      }}
+    </TooltipContainer>
+  );
+};
+
+Default.story = {
+  name: 'useTooltip'
+};
+
+WithPopper.story = {
+  name: 'positioning via Popper.js'
+};
+
+FocusableTooltip.story = {
+  name: 'with focusable tooltip'
+};
+
+Container.story = {
+  name: 'TooltipContainer'
+};
+
+export default {
+  title: 'Tooltip Container',
+  decorators: [withKnobs]
+};
```

Comment:
The hierarchy of the side-bar for Tooltip Container has changed slightly — it is now flatter because each of the four examples is a story that doesn't have nesting. I believe this aligns more with Storybook given how Storybook defines a "story" and I didn't find an easy way to set up the old hierarchy from the [Story hierarchy docs](https://storybook.js.org/docs/basics/writing-stories/#story-hierarchy).

**Before**:

* Tooltip Container
  - useTooltip
    - default
    - positioning via Popper.JS
    - with focusable tooltip
* TooltipContainer

**After**:

* Tooltip Container
  - useTooltip
  - positioning via Popper.JS
  - with focusable tooltip
  - TooltipContainer